pyproject.toml

```
@@ -11,7 +11,7 @@ authors = [{name = "Python Language Server Contributors"}]
 description = "Python Language Server for the Language Server Protocol"
 readme = "README.md"
 license = {text = "MIT"}
-requires-python = ">=3.8"
+requires-python = ">=3.0"
```

Seems like a typo happened here. This needs to be `>=3.9`:

```suggestion
requires-python = ">=3.9"
```